Specs/test.cjs

```diff
@@ -1,25 +1,30 @@
 /*eslint-env node*/
 "use strict";
 
-// NodeJS smoke screen test
-
 const assert = require("node:assert");
 const {
   Cartographic,
   createWorldTerrainAsync,
   sampleTerrain,
 } = require("cesium");
 
+// NodeJS smoke screen test
+
 async function test() {
+  console.log("start");
   const provider = await createWorldTerrainAsync();
+  console.log("loaded terrain");
   const results = await sampleTerrain(provider, 11, [
     Cartographic.fromDegrees(86.925145, 27.988257),
     Cartographic.fromDegrees(87.0, 28.0),
   ]);
+  console.log("sampled terrain");
   assert(results[0].height > 5000);
   assert(results[0].height < 10000);
   assert(results[1].height > 5000);
   assert(results[1].height < 10000);
+
+  console.log("all assertions passed");
```

Comment:
Do you have a specific reason? I feel like there should be an easy way to specifically see when it's successful. When there is no output it's not easy to tell when there was an invisible crash/failure vs a success. I'm open to reducing the number of them or changing what they say but I think it should output _something_.

Reply:
At the very least, it will clutter the CI logs, which we're already pretty bad about. It will fail on error if the code does not execute as excpected.

The fact that `test.cjs` it's not running on your system is a problem that we should look into, but that would a separate issue from this one.